src/MeadeCommandProcessor.cpp

```diff
@@ -1071,6 +1087,12 @@ String MeadeCommandProcessor::handleMeadeInit(String inCmd)
     _lcdMenu->printMenu("Remote control");
     _lcdMenu->setCursor(0, 1);
     _lcdMenu->printMenu(">SELECT to quit");
+
```

Comment:
The idea is that you don't want to heat until you're connected, in particular when you only start the mount.

Reply:
This is the opposite as to how the system has operated previously, it's common that people want it to start immediately since you could be setting up the OAT hours before astrodark to allow everything to acclimatize to ambient.

At the very least there needs to be a config option to start heating at boot using the previous EEPROM setting, or a configured default in the case of a cleared memory.

Reply:
Ah ok. I'll move it then. I normally have the thing sitting here turned on and wouldn't want to heat the guider, but I can just turn it off.